docs/business.yaml

```diff
@@ -258,6 +258,89 @@ paths:
                         priority: false
                         status: PENDING
                         submitter: mocked submitter
+                change-of-address-response:
```

Comment:
consider adding the status of this response (success/failed) to the example name. e.g. `consent-continuation-out-success-response`

Reply:
other than that, looking good 😄